src/colorprint.cc

```diff
@@ -89,7 +89,11 @@ std::string FormatString(const char* msg, va_list args) {
 
   std::size_t size = 256;
   char local_buff[256];
+#if defined(__ANDROID__) && (defined(__GLIBCXX__) || defined(__GLIBCPP__))
```

Comment:
Let's just use `vsnprintf` unconditionally. I see no reason to prefer the version in `std::` .